src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -47,6 +47,11 @@ internal enum ConversionKind : byte
         // IdentityValue is not a part of the language. 
         // It is used by lowering to ensure that trivially reduced expressions 
         // do not become exposed to mutations if used as receivers of struct methods.
-        IdentityValue,  
+        IdentityValue,
+
+        // ObjectToPointer is not directly a part of the language
+        // It is used by lowering of "fixed" statements to represent conversion of an object reference (O) to an unmanaged pointer (*)
+        // The conversion is unsafe and makes sense only if (O) is pinned.
+        ObjectToPointer,
```

Comment:
Converting a pinned object local. Regular local, not byref one. LIke if you have a pinned string.

---
In reply to: [124912940](https://github.com/dotnet/roslyn/pull/20548#discussion_r124912940) [](ancestors = 124912940)